src/ostree/ot-builtin-find-remotes.c

```diff
@@ -30,12 +30,14 @@
 #include "ostree-remote-private.h"
 
 static gchar *opt_cache_dir = NULL;
+static gchar *opt_finders = NULL;
 static gboolean opt_disable_fsync = FALSE;
 static gboolean opt_pull = FALSE;
 
 static GOptionEntry options[] =
   {
     { "cache-dir", 0, 0, G_OPTION_ARG_FILENAME, &opt_cache_dir, "Use custom cache dir", NULL },
+    { "finders", 0, 0, G_OPTION_ARG_STRING, &opt_finders, "Use the specified comma separated list of finders (e.g. config,lan,mount)", "FINDERS" },
```

Comment:
The way it's written right now if you specify `--finders=lan` when ostree doesn't have Avahi support it errors out, but if you don't use `--finders` it just doesn't use the LAN finder. So in that sense the default behavior isn't exactly the same as `--finders=config,lan,mount` so it wouldn't be quite accurate to say `default: all` unless `--finders=all` and `--finders=config,lan,mount` are considered to be different (and maybe that's ok). I think for now I'll just leave it.

Reply:
Yeah, I would consider `--finders=all` to be equivalent to `--finders=config,lan,mount` if Avahi support is available, and equivalent to `--finders=config,mount` otherwise. i.e. `all` should be ‘all supported and available finders’.